config/v1/types_infrastructure.go

```diff
@@ -157,12 +209,45 @@ type PlatformStatus struct {
 	IBMCloud *IBMCloudPlatformStatus `json:"ibmcloud,omitempty"`
 }
 
+// AWSServiceEndpoint store the configuration of a custom url to
+// override existing defaults of AWS Services.
+type AWSServiceEndpoint struct {
+	// Name is the name of the AWS service.
+	// This must be provided and cannot be empty.
+	Name string `json:"name"`
+
+	// URL is hostname only or fully qualified URI that overrides the default generated
```

Comment:
do we allow http or only https?  I strongly prefer the latter.

Reply:
> personally that's the restriction from AWS SDK, and i'm tracking that here. it doesn't say http or https only...

We get to be opinionated in our API.  If calls should only be https, then we should restrict them.  The doc doesn't match the field name.  why hostname only in a `url` field?